.luarc.json

```diff
@@ -1,6 +1,12 @@
 {
   "$schema": "https://raw.githubusercontent.com/sumneko/vscode-lua/master/setting/schema.json",
   "runtime.version" : "Lua 5.1",
+  "workspace": {
+    "library": [
+      "$VIMRUNTIME",
+      "${3rd}/luv/library"
+    ]
+  },
   "diagnostics": {
     "globals": [
       "vim"
```

Comment:
We can't remove this just yet as there are some warnings, although they may be spurious.